buildWindows.bat

```diff
@@ -1,50 +1,28 @@
 @echo off
+setlocal
 
-call "%PROGRAMFILES(X86)%\Microsoft Visual Studio 14.0\vc\vcvarsall.bat" x86
+call "C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Auxiliary\Build\vcvarsall.bat" x86
```

Comment:
`%PROGRAMFILES(X86)%` didn't work?
(Not a big deal, I see it hard-coded in other lua build files too anyway)